tests/boot/uefi_bootmenu.pm

```diff
@@ -21,7 +21,7 @@ sub run {
     my ($self) = @_;
     # because of broken firmware, bootindex doesn't work on aarch64 bsc#1022064
     # selecting a workaround is handled in boot/boot_to_desktop
-    return if (get_var('MACHINE') =~ qr'aarch64' && get_var('BOOT_HDD_IMAGE'));
+    return if (check_var('ARCH', 'aarch64') && get_var('BOOT_HDD_IMAGE'));
```

Comment:
I think what you are doing here is wrong, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2877#issuecomment-301412694 and my previous comment. Also the infile comments in line 22-23 clearly say "because of broken firmware", not "because aarch64 is a broken architecture".

@ldevulder please change back or explain with a good reason why the previous approach does not apply anymore